src/main/java/seedu/Equipment/EquipmentManager.java

```diff
@@ -0,0 +1,39 @@
+package seedu.Equipment;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+
+public class EquipmentManager {
+    private final HashMap<String, Equipment> equipmentList = new HashMap<>();
+
+    public void addEquipment(String itemName, String serialNumber, EquipmentType type, double cost, String purchasedFrom, String purchasedDate) {
+        Equipment equipment = new Equipment(itemName, serialNumber, type, cost, purchasedFrom, purchasedDate);
+        equipmentList.put(serialNumber, equipment);
```

Comment:
if the serialNumber already exists, should throw an error that will be caught when AddCommand.execute() is run